tests/angular-localForage.js

```diff
@@ -465,4 +477,50 @@ describe('Module: LocalForageModule', function() {
       }, done);
     });
   });
+
+  describe("instance", function () {
+      var testInstance_1;
+      var testInstance_2;
+      it('should create new instance for a storeName', function () {
```

Comment:
Some of these cases are really tests for `createInstance`, which we should have, but don't. You're welcome to move them into their own describe block and write real test cases around `createInstance` since this is relevant to the PR. For testing `instance`, will you just move a few of these cases into a `beforeEach` to set up the state of the test, and then only `it` the things that describe the `instance` method? Like:

```js
beforeEach(function () {
  testInstance_1 = $localForage.createInstance({
    name: 'testName',
    storeName: 'TEST_INSTANCE_1'
  });
  testInstance_2 = $localForage.createInstance({
    name: 'testName',
    storeName: 'TEST_INSTANCE_2'
  });
});

it('retrieve the instance by name and storeName', function () {
  expect($localForage.instance({
    name: 'testName',
    storeName: 'TEST_INSTANCE_1'
  })).toBe(jasmine.any(LocalForageInstance));
});
```